index.d.ts

```diff
@@ -1552,7 +1552,7 @@ declare class Tray {
 
 	/**
 	 */
-  has(key): boolean;
+	has(key): boolean;
```

Comment:
What's up with these indentation changes in this file ?

Reply:
99% of the file was using tabs, so my local linter changed the remaining few lines with spaces to also have tabs. I can change all the lines to spaces if you would prefer, that would be no issue. Not sure why there was originally a mix, I do not believe that I introduced that.